ManagementPacks/Community.DataOnDemand.Unix/Modules/Probe.GetNetstatCsv.mpx

```diff
@@ -0,0 +1,70 @@
+﻿<ManagementPackFragment SchemaVersion="2.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <TypeDefinitions>
+    <ModuleTypes>
+      <ProbeActionModuleType ID="Community.DataOnDemand.Unix.Probe.GetNetstatCsv" Accessibility="Public" Batching="false">
+        <Configuration>
+          <xsd:element name="Format" minOccurs="1" maxOccurs="1">
+            <xsd:simpleType>
+              <xsd:restriction base="xsd:string">
+                <xsd:enumeration value="csv" />
+                <xsd:enumeration value="csvEx" />
+              </xsd:restriction>
+            </xsd:simpleType>
+          </xsd:element>
+          <xsd:element name="TargetSystem" type="xsd:string" minOccurs="1" maxOccurs="1" />
+          <xsd:element name="TimeoutSeconds" type="xsd:integer" minOccurs="1" maxOccurs="1"/>
+        </Configuration>
+        <OverrideableParameters>
+          <OverrideableParameter ID="Format" Selector="$Config/Format$" ParameterType="string"/>
+          <OverrideableParameter ID="TimeoutSeconds" Selector="$Config/TimeoutSeconds$" ParameterType="int" />
+        </OverrideableParameters>
+        <ModuleImplementation>
+          <Composite>
+            <MemberModules>
+              <ProbeAction ID="PassThru" TypeID="System!System.PassThroughProbe">
+              </ProbeAction>
+              <ProbeAction ID="Probe" TypeID="MUL!Microsoft.Unix.WSMan.Invoke.Privileged.ProbeAction">
+                <TargetSystem>$Config/TargetSystem$</TargetSystem>
+                <Uri>http://schemas.microsoft.com/wbem/wscim/1/cim-schema/2/SCX_OperatingSystem?__cimnamespace=root/scx</Uri>
+                <Selector/>
+                <InvokeAction>ExecuteScript</InvokeAction>
+                <Input>
+                  <![CDATA[<p:ExecuteScript_INPUT xmlns:p="http://schemas.microsoft.com/wbem/wscim/1/cim-schema/2/SCX_OperatingSystem">
+                    <p:script>$IncludeFileContent/Scripts/GetNetstatCSV.sh$</p:script>
+                    <p:arguments>$Config/Format$</p:arguments>
+                    <p:timeout>$Config/TimeoutSeconds$</p:timeout>
+                  </p:ExecuteScript_INPUT>]]>
+                </Input>
+              </ProbeAction>
+            </MemberModules>
+            <Composition>
+              <Node ID="Probe">
+                <Node ID="PassThru" />
+              </Node>
+            </Composition>
+          </Composite>          
+        </ModuleImplementation>
+        <OutputType>MSWL!Microsoft.SystemCenter.WSManagement.WSManData</OutputType>
+        <TriggerOnly>true</TriggerOnly>
+      </ProbeActionModuleType>
+    </ModuleTypes>
+  </TypeDefinitions>
+  <LanguagePacks>
+    <LanguagePack ID="ENU" IsDefault="true">
+      <DisplayStrings>
+        <DisplayString ElementID="Community.DataOnDemand.Unix.Probe.GetNetstatCsv">
+          <Name>Data On Demand GetNetstatCSV Probe</Name>
+          <Description>Displays established TCP connections collected via netstat -ano</Description>
```

Comment:
I'll correct this, as those are the windows command line options.  Should be -tpn